### DISCUSSION TITLE
Pimcore Academy: Working with Objects Advanced - What statements concerning fieldcollections and blocks are correct?

### DISCUSSION THREAD

**Discussion by patriciomg** [pimcore/pimcore#15924] [Q&A / Support] [answered]
Hi everyone,
maybe someone can help me to understand why this answers are incorrect. 

![q9](https://github.com/pimcore/pimcore/assets/1974038/dae87d10-cc54-4a01-b7a0-40b0aaee12ff)

thanks in advance.

**Comment by kingjia90** [comment, marked as answer]
`block definitions can be reused in multiple class definitions` is wrong, Block is only used on the same class definition. 
One can copy the same config tree, but it's not shared among the class definitions and one change in one block is not synced to the others.

Pro tip: to remember it easily, only the types appearing here like Fieldcollection, Object Brick and Classification Store can be defined "externally" from the class definition and therefore be re-used in multiple class definitions.
![Screenshot from 2023-09-11 17-47-31](https://github.com/pimcore/pimcore/assets/6014195/bca94646-6e01-47d2-9968-8fb025209be4)

**Reply by patriciomg** [reply]
aahh now I see it, thank you very much :)
And thank you for the pro tip too 👍 

cheers!
